tests/test_searching.py

```diff
@@ -0,0 +1,44 @@
+"""Module to test the Sorting algorithms."""
+
+
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from algorithms.searching.search import Search
+from algorithms.searching.lineal_search import LinealSearch
+from algorithms.searching.binary_search import BinarySearch
```

Comment:
I don't see any `binary_search` file, so I assume this will fail, though I haven't run this locally. Maybe don't add it now. We should make a separate PR for the Binary Search and we can think about how to implement it better.